src/App.css

```diff
@@ -1,19 +1,20 @@
 .App {
-  display: grid;
-  grid-template-rows: min-content 1fr;
-  gap: var(--fixed-spacing--1x);
-  padding: var(--fixed-spacing--1x);
-
   position: absolute;
   inset: 0;
+
+  padding: var(--fixed-spacing--1x);
+
+  display: grid;
+  grid: min-content 1fr / auto;
+  gap: var(--fixed-spacing--1x);
```

Comment:
This order of properties moves from "how I am positioned relative to the page / my parent" to "how things outside of my content are styled" to "how my children are positioned/behave". I've no idea why this suddenly mattered to me.